.github/memory_statistics_config.json

```diff
@@ -1,9 +1,11 @@
 {
     "lib_name": "AWS IoT Jobs",
     "src": [
-        "source/jobs.c"
+        "source/jobs.c",
+        "coreJSON/source/core_json.c"
```

Comment:
What's the reason that we added the file size check to `core_json.c` inside of this repo?

Reply:
The functions added to the repo have a dependency on coreJSON, so that is the reason. However, you are correct and coreJSON should be added to `.gitmodules` and pulled in that way.

Reply:
Speaking with @archigup and @kstribrnAmzn, we are trying to move away from using submodules within libraries, and are moving towards using FetchContent with cmake.

Reply:
It doesn't make sense to include the size check of coreJSON's files inside of this repo.

The size of that file should be checked and maintained in coreJSON, not in here.